controllers/news.controllers.js

```diff
@@ -44,4 +45,10 @@ exports.addVotes = (req, res, next) => {
     const uVotes = req.body.inc_votes
     insertVotes(article_id, uVotes).then((patchedArticle)=> res.status(201).send({patchedArticle}))
     .catch(next)
+}
+
+exports.removeComment = (req, res, next) =>{
+    const {comment_id} = req.params
+    deleteComment(comment_id).then((removedData)=> res.status(204).send(removedData))
```

Comment:
204 never sends anything back - so there's no need to include `removedData` - you can just do `.send()`